<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandmarksExporterWidget</class>
 <widget class="qSlicerWidget" name="LandmarksExporterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="fiducialSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of the fiducial computed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLAnnotationHierarchyNode</string>
      </stringlist>
     </property>
     <property name="noneEnabled">
      <bool>false</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="applyButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export fiducial list into the output file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="fiducialListSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of the fiducial computed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Fiducial list</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="outputFileLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;File where the output will be computed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Output file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkPathLineEdit" name="outputPath">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;File where the output will be computed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="filters">
      <set>ctkPathLineEdit::Files|ctkPathLineEdit::NoDotAndDotDot|ctkPathLineEdit::Readable</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>LandmarksExporterWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>fiducialSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
